docs/integrations/google/compute-engine.md

```diff
@@ -314,6 +314,8 @@ See the details of projects, zones, instances, and message severity in your Goog
 
 See the details of message severities - emergency, alert, critical, error, and warning.
 
+<img src={useBaseUrl('img/integrations/google/google-compute-engine-severity.png')} alt="Google Compute Engine dashboards" />
```

Comment:
Image is not added. It is broken when previewed, kindly check.

Reply:
adding the image